eng/pipelines/docindex.yml

```diff
@@ -10,6 +10,7 @@ jobs:
       DocRepoLocation: $(Pipeline.Workspace)/docs
       DocRepoOwner: MicrosoftDocs
       DocRepoName: azure-docs-sdk-node
+      ImageId: azuresdkimages.azurecr.io/jsrefautocr:latest
```

Comment:
We should name this DocValidationImageId in all the places in this file as well, just to help clarify which image this is.

Reply:
We have `pull-docker-image.yml` which I think it is reasonable to keep `ImageId` as general.
The other place `Update-DocsMsPackage` has `ImageId` as parameter name. It is in common folder. I am not able to change in this PR. Need a followup one.

I changed the name for all other places.